library/modules/Gui.cpp

```diff
@@ -1624,6 +2043,71 @@ df::coord Gui::getCursorPos()
     return df::coord(cursor->x, cursor->y, cursor->z);
 }
 
+void Gui::recenterViewscreen(int32_t x, int32_t y, int32_t z, df::report_zoom_type zoom)
```

Comment:
Is this function meaningfully different from `Gui::revealInDwarfmodeMap(..., center=true)`? I expect there are some slight behavioral differences (e.g. with the separate `Item` vs. `Unit` modes) but does DFHack need two slightly different ways to center the viewport?

Reply:
Was going to handle this in a different PR, where the code looks like this:
```
bool Gui::revealInDwarfmodeMap(df::coord pos, bool center)
{
    using df::global::window_x;
    using df::global::window_y;
    using df::global::window_z;

    if (!window_x || !window_y || !window_z || !world)
        return false;
    if (!Maps::isValidTilePos(pos))
        return false;

    recenterViewscreen(pos, (center ? report_zoom_type::Unit : report_zoom_type::Item));

    return true;
}
```

Should I rename ``recenterViewscreen`` to ``revealInDwarfmodeMap``?